backend/server/public/src/reducers.js

```diff
@@ -75,6 +77,17 @@ export const ad = (state = {}, action) => {
   }
 };
 
+export const groupedAttribute = (state = [], action) => {
+  switch (action.type) {
+    case GOT_RECENT_GROUPED_ATTR:
+      return action.groupedAttrs;
+    case REQUESTING_RECENT_GROUPED_ATTR:
```

Comment:
Sorta weird to have an action that doesn’t change any state. Do we need it?

Reply:
I'll add the word "loading"

Reply:
done in 8f4aa31